.github/workflows/deploy.yml

```diff
@@ -0,0 +1,17 @@
+name: ci
```

Comment:
Would likely change this or the other name so it's easier to differentiate in actions

Reply:
Good call, done!